soundcrate/src/lib/models/user.js

```diff
@@ -15,7 +14,8 @@ const userSchema = new Schema({
     },
     imageUrl: {
       type: String,
-      required: false
+      required: true,
+      default: "./images/default-user.png",
```

Comment:
Nit: I still think it makes more sense to not store an `imageUrl` field at all unless the user provides one since the this schema currently would necessitate that the default image remains at the same path so we wouldn't be able to potentially change that without changing all the database entries.

Reply:
okay that makes sense

Reply:
If you've already done the frontend to work specifically with this then let's keep as-is!